docs/translation.md

```diff
@@ -66,6 +66,26 @@ Scenario('Efetuar login', (Eu) => {
 });
 ```
 
+## French
+
+To write your tests in French you can enable the French translation by adding to config:
+
+```json
+  "translation": "fr-FR"
+```
+
+Now you can write tests like this:
+
+```js
+Fonctionnalité('Se connecter sur GitHub', (io) => {
```

Comment:
Oups, I just figured out that this example is not correct.
According to the translation file, it should be:

```js
Scénario('Se connecter sur GitHub', (io) => {
```

@vimar could you quick fix this?

Reply:
Also, why it is `io` if you use `Ja` ?

Reply:
A copy/paste error :disappointed: 
I fix this in the new pr: https://github.com/codeceptjs/CodeceptJS/pull/2578